index.d.ts

```diff
@@ -1,4 +1,5 @@
-import type { ApolloServerPlugin } from "apollo-server-plugin-base";
+import type {ApolloServerPlugin as Base } from "apollo-server-plugin-base";
```

Comment:
Ultra-nitpicky: will eslint complain if there's not a space after the left brace?

Reply:
Yea i can update. it's prob because eslint isn't running on our types, i'll update that